_episodes/05-merging-data.md

```diff
@@ -11,7 +11,10 @@ objectives:
     - "Employ `to_csv` to export a DataFrame in CSV format."
     - "Join DataFrames using common fields (join keys)."
 keypoints:
-    - "FIXME"
+    - "`Merge` and `concat` can be used to recombine subsets from a DataFrame, or even data from different files."
```

Comment:
Python is case-sensitive, so... maybe update to:
> "Panda's `.merge` and `.concat` methods are used to (re-)combine subsets of a DataFrame or even data from different files."

Reply:
```suggestion
    - "`merge` and `concat` can be used to (re)-combine subsets from a DataFrame, or even data from different files."
```

I left them in without the dot notation, but lowercase. There are other points where we refer to methods without dot notation.

Reply:
Yeah, I'm fine with that. I think it _might_ be useful to use the same approach across multiple lessons but this is not important at this point.

Reply:
shall it, perhaps, be `combine subsets of a DataFrame` ?

Reply:
```suggestion
    - "`Merge` and `concat` can be used to combine subsets of a DataFrame, or even data from different files."
```